networkd/network_server.ml

```diff
@@ -122,7 +123,9 @@ let need_enic_workaround () =
 		| None -> false )
 
 module Sriov = struct
```

Comment:
This module should be moved down after the PVS_proxy module, there should be already an `S` module defined there: https://github.com/xapi-project/xcp-networkd/blob/master/networkd/network_server.ml#L1009

Reply:
...or remove the definition of `S` below in favour of the one above. I would suggest to not move the module in this PR, because it causes unnecessary conflicts in another PR, and this PR isn't actually introducing the module.

Reply:
We looked into this and it looks like the modules S was moved in different commits, and the change went lost in the rebase.

Reply:
Also, the module needs to be where it is, because it is used in `get_capabilities`: 
https://github.com/minli1/xcp-networkd/blob/3a9e6d460deafdbce66a211cbf479f1723673666/networkd/network_server.ml#L552

Again, this is an older change that got hidden by the rebase